samples/shields/npm6001_ek/doc/index.rst

```diff
@@ -41,6 +41,68 @@ Note that this sample automatically sets ``SHIELD`` to ``npm6001_ek``. Once
 flashed, you should boot into the shell interface. The ``npm6001`` command is
```

Comment:
The "Overview" section of this file has a list of features functionalities supported by this sample. "Regulator" is missing from this list

Reply:
added, thanks for spotting